upstairs/src/test.rs

```diff
@@ -1560,40 +1576,12 @@ mod up_test {
         assert!(ds.downstairs_errors.get(&0).is_some());
         assert!(ds.downstairs_errors.get(&1).is_some());
         assert!(ds.downstairs_errors.get(&2).is_none());
-
-        // another read. Make sure only client 2 returns data.
```

Comment:
why was this removed?

Reply:
With the new fault behavior, this test does not have enough framework to move IOs forward.
The marking of a downstairs as faulted now happens in `process_ds_operation` instead of
`process_ds_completion`.  New tests added now cover what was covered in this test before.